slack-buggybot/commands/next.rb

```diff
@@ -28,16 +28,20 @@ def self.call(client, data, match)
         case match[:expression]
         when 'fixed'
           # Nop, it's the default
+          emoji = ':party-cat:'
         when 'docs'
           fate = 'added_docs'
           message = 'added docs to'
+          emoji = ':writing_hand:'
         when 'verified'
           fate = 'verified'
           message = 'verified the existence of'
+          emoji = ':110:'
         when 'interlinked'
           fate = 'interlinked'
           message = 'added context to'
-        when 'none'
+          emoji = ':link:'
+        when 'skip'
```

Comment:
`@buggy next skip` seems more intuitive than `@buggy next none`

Reply:
Yup, makes sense. There’s one mention of none left in the readme but I can fix after.